docs/whats-new/2021-04.md

```diff
@@ -0,0 +1,93 @@
+---
+title: ".NET docs: What's new for April 2021"
+description: "What's new in the .NET docs for April 1, 2021 - April 30, 2021."
+ms.date: 05/03/2021
+---
+
+# .NET docs: What's new for April 2021
+
+Welcome to what's new in the .NET docs from April 1, 2021 through May 1, 2021. This article lists some of the major changes to docs during this period.
+
+## .NET Framework
+
+### Updated articles
+
+- [Performance counters in .NET Framework](../framework/debug-trace-profile/performance-counters.md) - Fix ms.topic for debug-trace-profile folder
+- [How to: Configure .NET Framework-Based COM Components for Registration-Free Activation](../framework/interop/configure-net-framework-based-com-components-for-reg.md) - Set ms.topic to how-to
+- [How to: Implement a Client Application that Uses the Discovery Proxy to Find a Service](../framework/wcf/feature-details/client-app-discovery-proxy-to-find-a-service.md) - Set ms.topic to how-to
+
+## .NET fundamentals
+
+### New articles
+
+- [SYSLIB0013: EscapeUriString is obsolete](../core/compatibility/syslib-warnings/syslib0013.md) - Add new syslib warnings
```

Comment:
I thought you changed the tool not to include PR titles for new articles. Maybe you just removed the titles manually?

Reply:
Yeah, I did it by hand. I'll update the code for this month.